src/state/tables.ts

```diff
@@ -15,16 +15,16 @@ import {
   Task,
 } from '../types';
 import { Loadable, LOADING_STATE, useCachedLoadable } from './common';
-import { $postgresqlStatus } from './services';
+import { $metabaseStatus } from './services';
 
 // State
 
 const $tableListInvalidator = atom(0);
 
 const $tableList = abortableAtom((get, { signal }) => {
   get($tableListInvalidator);
-  const postgresqlStatus = get($postgresqlStatus);
-  if (postgresqlStatus === ServiceStatus.Running) {
+  const metabaseStatus = get($metabaseStatus);
+  if (metabaseStatus === ServiceStatus.Running) {
```

Comment:
We actually need both to be ready.

Reply:
oh, I figured mb is dependent on pg, but you're right, that's the case only in the startup scenario.